mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/DaoUpdateMethodGenerator.java

```diff
@@ -119,13 +120,22 @@ public DaoUpdateMethodGenerator(
     List<? extends VariableElement> parameters = methodElement.getParameters();
     String entityParameterName = parameters.get(0).getSimpleName().toString();
 
-    String customWhereClause = methodElement.getAnnotation(Update.class).customWhereClause();
+    Update annotation = methodElement.getAnnotation(Update.class);
+    String customWhereClause = annotation.customWhereClause();
+    NullSavingStrategy nullSavingStrategy = annotation.nullSavingStrategy();
+
+    methodBuilder.addStatement(
+        "$1T nullSavingStrategy = $1T.$2L", NullSavingStrategy.class, nullSavingStrategy);
     if (customWhereClause.isEmpty()) {
       // We generated an update by primary key (see maybeAddWhereClause), all entity properties are
       // present as placeholders.
       methodBuilder.addStatement(
-          "$L.set($L, boundStatementBuilder)", helperFieldName, entityParameterName);
+          "$1L.set($2L, boundStatementBuilder, $3L)",
```

Comment:
Nit: you can hardcode `nullSavingStrategy` here since it's constant
```suggestion
          "$1L.set($2L, boundStatementBuilder, nullSavingStrategy)",
```